.gitignore

```
@@ -3,4 +3,5 @@
 .vagrant
 /phpunit.xml
 .phpunit
+.phpunit.result.cache
```

```suggestion
*.cache
```

Then we can also remove the next line `/.php_cs.cache`.